src/yb/master/master-path-handlers.cc

```diff
@@ -1548,11 +1548,15 @@ void MasterPathHandlers::CalculateTabletMap(TabletCountMap* tablet_map) {
     bool is_user_table = master_->catalog_manager()->IsUserCreatedTable(*table);
 
     for (const auto& tablet : tablets) {
+      if (master_->catalog_manager()->IsColocatedUserTable(*table)) { // will be taken care of by colocated parent table
```

Comment:
Also put this one level outwards at the tables for loop.

Reply:
Done